README.md

```diff
@@ -0,0 +1,50 @@
+# libholiday
+
+[![Build Status](https://travis-ci.org/mayflower/libholiday.svg?branch=master)](https://travis-ci.org/mayflower/libholiday)
+
+A library to calculate holidays.
+
+Supported countries and states
+
+* America
+* Germany and all constituent states
+* Sweden
+* Netherlands
+
+Usage
+
+Add to your projects ```composer.json```
```

Comment:
`composer require` is more common now, I think...

Reply:
yes but how to require from github?

Reply:
Ok, done. :+1: